ci-operator/config/red-hat-data-services/openvino_model_server/red-hat-data-services-openvino_model_server-v2022.3-release.yaml

```diff
@@ -0,0 +1,216 @@
+# binary_build_commands: NVIDIA=1 RUN_TESTS=0 BUILDAH_FORMAT=docker OV_USE_BINARY=0
+#   BASE_OS=redhat RH_ENTITLEMENT=1 RH_ENTITLEMENT_DIR=$(pwd)/entitlement OV_SOURCE_BRANCH='releases/2022/3'
+#   OV_CONTRIB_BRANCH='releases/2022/3' make docker_build
+
+# The above invokes docker build commands, is that possible? If not, the below replicates most of what the make command does
+
+build_root:
+  from_repository: true # Need some clarification for this part
+
+# Entitlements, is the cluster able to build with entitlements or no? Need to figure that out since there's 4 or 5 packages not in UBI repos that we need from RHEL/CodeReady Builder
+
+images: # Does this allow dependencies?
+  - dockerfile_literal: |- # Guessing this is how one would do the equivalent of a shell cp in the source directory?
+      FROM ubi8/ubi
+      COPY src/queue.hpp /
+    inputs:
+      bin:
+        paths:
+          - destination_dir: /src/custom_nodes/
+            source_path: /queue.hpp
+    to: "shell-commands:1"
+  - dockerfile_path: src/custom_nodes/Dockerfile.redhat
+    context_dir: src/custom_nodes # Guessing the name of this field from https://github.com/openshift/ci-tools/blob/91f8a8f09c57e637223ca312cdc454a08a305fd7/pkg/api/types.go#L1843
+    inputs: # It says inputs, but https://github.com/openshift/ci-tools/blob/91f8a8f09c57e637223ca312cdc454a08a305fd7/pkg/api/types.go#LL1992-L1993C23 seems to suggest that it's really copying stuff back out from the image? How do we supply files? COPY directive from the Dockerfile I'm guessing
+    # *** Need to add queue.hpp into the build images, how can we copy it into place? Should I just do that in the Dockerfile? ***
+      bin: # What is the "bin tag"?
+        paths: # This lets us copy stuff out of the image right? Does it happen at the beginning or end of the build? I'm assuming the end
+        - destination_dir: lib/redhat # Is this relative to context_dir or no?
+          source_path: /custom_nodes/lib/libcustom_node_add_one.so
+    build_args: # Basing this on https://github.com/openshift/ci-tools/blob/91f8a8f09c57e637223ca312cdc454a08a305fd7/pkg/api/types.go#L1968 right?
+      - name: NODE_NAME
+        value: add_one
+      - name: NODE_TYPE
+        value: cpp
+    to: "openvino/model_server-custom-node:add_one" # This seems to be the output image name, we can use this in future layers right?
```

Comment:
This is only the tag. That image is going to be used in your tests or when promoting based on the `promotion` configuration.

Reply:
Ok, so when promoting, we specify just what tag(s?) to promote